nebula_hw_interfaces/src/nebula_robosense_hw_interfaces/robosense_hw_interface.cpp

```diff
@@ -213,12 +213,8 @@ Status RobosenseHwInterface::GetLidarCalibrationFromSensor(
     uint8_t return_mode_data = info_buffer_.value()[HELIOS5515_RETURN_MODE_OFFSET];
     if (return_mode_data == 0x00) {
       return_mode = ReturnMode::DUAL;
-    } else if (return_mode_data == 0x04) {
-      return_mode = ReturnMode::STRONGEST;
-    } else if (return_mode_data == 0x05) {
-      return_mode = ReturnMode::LAST;
-    } else if (return_mode_data == 0x06) {
-      return_mode = ReturnMode::FIRST;
+    } else {
+      return_mode = ReturnMode::SINGLE;
```

Comment:
Could you explain the motivation behind combining all single-return modes?
The datasheet specifies that strongest/last/first exist, so if possible, this should be supported by the driver.
Also, currently the return type returned by `ReturnModeToReturnType` falls back to `ReturnType::UNKNOWN` for `ReturnMode::SINGLE` (should be `STRONGEST` / `LAST` / `FIRST` respectiely).

Reply:
Hello @mojomex , I have updated this part. Now, for each point a return type is specified like the Hesai driver.